packages/fiori/src/types/TimelineItemStatus.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * Available status types of a timeline item.
+ * @public
+ */
+enum TimelineItemStatus {
```

Comment:
Did you consider using `valueState`? It seems exactly the same. And also calling the property `valueState` instead of `status`. I'm not sure this would be right, but maybe it's worth thinking about

Reply:
After team discussion we decided to rename the `status` property to `state`, re-using the already existing `ValueState` enum.